### DISCUSSION TITLE
`apply_tts()` hangs

### DISCUSSION THREAD

**Discussion by debagos** [snakers4/silero-models#195] [Q&A] [answered]
When I try to call `model.apply_tts` a second time, the process just hangs forever...
Any Idea what I'm doing wrong?

**Comment by snakers4** [comment, marked as answer]
https://github.com/snakers4/silero-models/issues/183

**Comment by debagos** [comment]
Wow thanks :heart:
Adding the following code fixed my issue, thank you @snakers4!
```python
torch._C._jit_set_profiling_mode(False)
```
